### ISSUE TITLE
App

### ISSUE THREAD

**Issue by asltwh033** [pwn20wndstuff/Undecimus#2023]
**Describe the feature you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by SarKaa** [comment]
You do realise you have to fill out the main text bit, so that the dev actually know what you want
At least try and help people trying to help you a little bit

**Comment by eglacias** [comment]
This is spam, isn’t there a moderator to help ban these people and close these useless spam posts?

**Comment by SarKaa** [comment]
I agree, as well as stupid posts like this one, there are people asking for random stuff, like a jailbreak bypass.
I have been spending some of my free time scrolling through the issues and trying to help people, telling them to go to reddit or discord, but they don't listen, and start arguing. There needs to be some  kind of moderator. I remember there used to be 9 issues before unc0ver 5.0.0. was released. but now there are 73. I can't even imagine how many of those are repeats, and how many are purely stupid, and how many are irrelevant. There are so many stupid issues that we don't even see the serious ones

**Comment by SarKaa** [comment]
As well as those stupid issues I mentioned above, there are so so many people complaining that their devices are incompatible, and then saying thats an error. I tried telling some people that ios 12.4.1 to 12.4.7 is not supported, but in return they called me stupid. These people don't deserve to ask for help here, its pure stupidity, and pisses me off cos I appreciate pwn20wnd's work, but these people take everything for granted, and expect everyone to spoonfeed them, and then when they have to face facts themselves, they start blaming others.

**Comment by eglacias** [comment]
Well, I’ll continue to help others and not blame them, and see if I can calm them down. I have at least seen the owners of the thread delete some particularly offensive posts before as well as some spam
